docs/getting-started.txt

```diff
@@ -1,300 +1,338 @@
-#################
- Getting Started
-#################
+#########################
+ Quickstart for ML Users
```

Comment:
```suggestion
 Quickstart for Model Developers
```

Deep Learning, which is a subset of Machine Learning, is what we focused on but we also don't want to shut the door for machine learning developers. I'd call it model developers.